lib/screens/transfer/send/send_confirmation/send_confirmation_screen.dart

```diff
@@ -40,6 +41,7 @@ class SendConfirmationScreen extends StatelessWidget {
               if (pageCommand is ShowTransferSuccess) {
                 if (pageCommand.shouldShowInAppReview) {
                   InAppReview.instance.requestReview();
+                  settingsStorage.saveDateSinceLastAsked(DateTime.now().millisecondsSinceEpoch);
```

Comment:
dont access services in the UI. Move this to a bloc/usecase

Reply:
best I can do is the mapper since is where we know for sure if the app is going to ask or no